src/pages/api/token/pr/[id]/index.ts

```diff
@@ -109,22 +109,16 @@ export default async function handler(
 
       const txid = computeTxId(token);
       if (request.reusable) {
-         await createTokenInDb({ token, giftId: null }, txid);
```

Comment:
how come we don't need to call this create token anymore?

Reply:
with `addTokenToPaymentRequest` we are still creating it in the db, but we are creating it and then connecting it to the payment request:

```ts
tokens: {
            create: {
               id: computeTxId(token),
               token,
            },
         },
  ```

The notification system is really wonky so right now the token notification has two references to the same token. One is the stringified token that is in the notification itself and the other is a reference to the Token in the db